templates/flutter/lib/src/client_browser.dart.twig

```diff
@@ -71,17 +77,20 @@ class ClientBrowser extends ClientBase with ClientMixin {
     return this;
   }
 
+  @override
   ClientBrowser setEndPointRealtime(String endPoint) {
     _endPointRealtime = endPoint;
     return this;
   }
```

Comment:
on line 68 you set `_endPointRealtime` when `Endpoint` is set, should we do vice-versa here?

Reply:
@wess I didn't change that line. All I changed here is adding the `@override` annotation.

Reply:
@wess we don't want to do that, not doing that in other SDKs as well (https://github.com/aw-tests/sdk-for-web/blob/c71bb37fcf33094752885397fb7fe3427696a1af/src/sdk.ts#L822) Just because, realtime can have separate endpoint than the main endpoint, right now we are using traefik to make realtime available via same endpoint /realtime but that might change, and one can directly access via different port as well.